p2p/service.go

```diff
@@ -667,7 +667,7 @@ func NewService(cfg *config.Config, events *event.Feed, param *params.Params) (*
 	}
 
 	s.host = h
-
+	param.CoinbaseVersionConfig.SetNodeInfo(h.ID().String())
```

Comment:
the code is Intrusive,  you should depend on the p2p.api , instead of change the behavior internally.